package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "ddq-mysql",
+    "version": "1.0.0",
+    "description": "A backend plugin that DeDuplicated Queue (DDQ) can use to communicate with a MySQL database",
+    "main": "index.js",
+    "scripts": {
+        "test": "istanbul cover jasmine-node --captureExceptions spec/ && codecov && eslint .",
+        "watch": "jasmine-node lib spec --autotest --watch"
+    },
+    "author": "DDQ Team",
+    "contributors": [
+        {
+            "name": "Chris Tetreault",
+            "url": "https://github.com/AbsentSemicolon"
+        },
+        {
+            "name": "Lucas Nagle",
```

Comment:
Probably doesn't make a different but you should be first. I look at this like a book and the first author is the more important one, but if we're need to do it alphabetically, then this would be correct. I would get another opinion.